main.go

```diff
@@ -8,17 +8,20 @@ import (
 	"net/http"
 	"os"
 	"runtime"
-	"sync"
 	"time"
 
-	"github.com/ericchiang/k8s"
 	apiv1 "github.com/ericchiang/k8s/api/v1"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 var (
-	addr = flag.String("listen-address", ":9101", "The address to listen on for HTTP requests.")
+	// flags
+	DrainNodeTimeout        = flag.Int("drain-node-timeout", 300, " Max time in second to wait before deleting a node.")
```

Comment:
The prometheus team has switched - at least for blackbox exporter - to https://github.com/alecthomas/kingpin for handling startup parameters. I've used it in the past, it's a pretty neat library.